selfdescribe.json

```diff
@@ -24780,6 +24782,18 @@
           "group": null,
           "default": true
         },
+        "pod_ephemeral_storage_capacity_bytes": {
+          "type": "gauge",
+          "description": "Represents the storage space available (bytes) for the filesystem",
```

Comment:
This is a pretty weak description.  What exactly is the ephemeral storage for a pod?  Is it the same as the root filesystem of the containers or is it something that you have to configure specially?

Reply:
It's a combination of the root filesystem for containers and empty dir backed volumes (https://kubernetes.io/docs/concepts/storage/volumes/#emptydir)

Reply:
I'll update the description with some more details

Reply:
Ok, yeah any links to official K8s docs would be good too.  Like the doc that says that it also includes emptyDir volumes.

Reply:
Updated metric descriptions.